web/src/main/java/org/cbioportal/web/StudyViewController.java

```diff
@@ -292,24 +318,23 @@
                     if (attributeDatatypeMap.containsKey(attribute.getAttributeId())) {
                         ClinicalDataType clinicalDataType = attributeDatatypeMap.get(attribute.getAttributeId());
                         List<String> filteredIds = clinicalDataType == ClinicalDataType.PATIENT
-                                ? filteredUniquePatientKeys
-                                : filteredUniqueSampleKeys;
+                            ? filteredUniquePatientKeys
+                            : filteredUniqueSampleKeys;
 
                         List<ClinicalDataBin> dataBins = dataBinner
-                                .calculateDataBins(attribute, clinicalDataType,
-                                        filteredClinicalDataByAttributeId.getOrDefault(attribute.getAttributeId(),
-                                                Collections.emptyList()),
-                                        filteredIds)
-                                .stream()
-                                .map(dataBin -> studyViewFilterUtil.dataBinToClinicalDataBin(attribute, dataBin))
-                                .collect(Collectors.toList());
+                            .calculateDataBins(attribute, clinicalDataType,
+                                filteredClinicalDataByAttributeId.getOrDefault(attribute.getAttributeId(),
+                                    Collections.emptyList()),
+                                filteredIds)
+                            .stream()
+                            .map(dataBin -> studyViewFilterUtil.dataBinToClinicalDataBin(attribute, dataBin))
+                            .collect(Collectors.toList());
                         clinicalDataBins.addAll(dataBins);
                     }
                 }
             }
         }
-
-        return new ResponseEntity<>(clinicalDataBins, HttpStatus.OK);
+        return clinicalDataBins;
```

Comment:
Just curious, don't we want to return like this?
```
return new ResponseEntity<>(clinicalDataBins, HttpStatus.OK);
```

Reply:
This change looks a bit misleading. We still do it in the `fetchClinicalDataBinCounts` method: https://github.com/cBioPortal/cbioportal/pull/8599/files/b0af01e571acaf7379703fb1b1fd6551c5169914#diff-79e9bada6b365f432f10c451b2b0eacbcbbd30fd0b8679fd2a1a4d0bb8bee921R189

Reply:
Yea, onur is right. The diff is just misleading. The controller method still returns a ResponseEntity.